src/app/prevention/strides-against/strides-against.component.ts

```diff
@@ -0,0 +1,249 @@
+import { Component, OnInit } from '@angular/core';
+import { ViewChild } from '@angular/core';
+import * as RaphaelJS from 'raphael';
+import { LanguageService } from '../../services/language.service';
+import { MatDialog } from '@angular/material';
+import { DialogComponent } from './dialog/dialog.component';
+import { Observable } from 'rxjs/Observable';
+import { SharedDataService } from '../../services/shared.data.service';
+import swal from 'sweetalert2';
+import { Router } from '@angular/router';
+import { BackgroundConfig, ObjectsConfig, ManConfig } from './game-config';
+import { DashboardService } from '../../services/dashboard.service';
+import { positionConfig } from './game-config';
+import { PositionElement } from './models/elements';
+import { Item } from './models/Item';
+
+export class PaperConfig {
+  imageIndex: number;
+  imagePositionIndex: number;
+}
+
+@Component({
+  selector: 'app-strides-against',
+  templateUrl: './strides-against.component.html',
+  styleUrls: ['./strides-against.component.scss']
+})
+export class StridesAgainstComponent implements OnInit {
+  // Initialising game area
+  @ViewChild('gameArea') gameArea;
+  paper: RaphaelPaper;
+  man: RaphaelElement;
+
+  language: any;
+  alerts: any;
+  pos = 0;
+  question: string;
+  result: any;
+
+  completed = false;
+  activityComplete = false;
+
+  // Stores the objects
+  elements: RaphaelElement[] = [];
+
+  answer: boolean;
+
+  // Configs:
+  BackgroundConfig = BackgroundConfig;
+  ObjectsConfig = ObjectsConfig;
+  ManConfig = ManConfig;
+  randomPosition: Array<PositionElement> = []; // After randomizing the original position config
+  positionConfig = positionConfig;
+
+  randomObjectConfig: Array<Item> = []; // After randomizing the original  objects config
+  paperConfig: Array<PaperConfig> = [];
+
+  // Changing this would change the no. of objects in the game
+  countImage = 5;
+
```

Comment:
Can we move this to a configuration file?

Reply:
Done!